Makefile

```diff
@@ -36,10 +36,16 @@ install:
 	install -m644 keyd.service $(DESTDIR)$(PREFIX)/lib/systemd/system
 	install -m755 bin/keyd $(DESTDIR)$(PREFIX)/bin
 	install -m644 keyd.1.gz $(DESTDIR)$(PREFIX)/share/man/man1
+install-usb-gadget:
+	install -m644 usb-gadget.service $(DESTDIR)$(PREFIX)/lib/systemd/system
```

Comment:
You should probably install this to `keyd-usb-gadget.service` (the same goes for the shell script).

Reply:
Too generic indeed. Renamed.